### PR TITLE
Fix README.md

### PR DESCRIPTION
**Pull request by son-vyas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
-# openstack-test-operator
\ No newline at end of file
+# openstack-tests-lists
+==========================
+
+Overview
+--------
+
+openstack-tests-list will generate a skip-list/allow-list to be executed by tempest.
```
Comment (kopecmartin) [inline README.md]: skip-list or skiplist? allow-list or allowlist? let's use one format

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
-# openstack-test-operator
\ No newline at end of file
+# openstack-tests-lists
```
Comment (kopecmartin) [inline README.md]: s/*-lists/*-list

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+
+openstack-tests-list will generate a skip-list/allow-list to be executed by tempest.
+Each component can manage their own skiplist/allowlist or can use default.
+
+-  Free software: Apache license
+
+Quickstart
+==========
+
+- edit tempest_skip.yml
```
Comment (kopecmartin) [inline README.md]: edit skiplist/allowlist 
(tempest_skip.yml is not mentioned, so far, in this context)

File: README.md [hunk 4]
```diff
@@ -1 +1,54 @@
...
+Installing from git and virtualenv::
+
+    $ git clone git@github.com:openstack-k8s-operators/openstack-tests-list.git
+    $ cd openstack-tests-list
+
+Examples
+--------
+
+A simple example of the structure of the yaml file expected by
+skiplist.
```
Comment (kopecmartin) [inline README.md]: .. structure of a skiplist yaml file expected by openstack-tests-list ?

File: README.md [hunk 5]
```diff
@@ -1 +1,54 @@
...
+          - 'undercloud'
+          - 'overcloud'
+        jobs:
+          - job1
+          - job2
+        reason: 'default reason'
+        releases:
+          - name: master
+
+In the above example, the test *full.tempest.test* will be executed by tempest.
```
Comment (kopecmartin) [inline README.md]: the "full.tempest.test" will be skipped/excluded as this is an example of a skiplist file.

File: README.md [hunk 6]
```diff
@@ -1 +1,54 @@
...
+It will also be skipped in the releases master, specifically
+in jobs job1 and job2. It will not be skipped in any other job, no matter what
+the release is.
+
+Removing the list of jobs, means it will be skipped everywhere.
```
Comment (kopecmartin) [inline README.md:54]: this sentence is true if we were talking about allow list, right?

File: README.md [hunk 7]
```diff
@@ -1 +1,54 @@
...
+          - 'overcloud'
+        jobs:
+          - job1
+          - job2
+        reason: 'default reason'
+        releases:
+          - name: master
+
+In the above example, the test *full.tempest.test* will be executed by tempest.
+It will also be skipped in the releases master, specifically
```
Comment (kopecmartin) [inline README.md]: it will be skipped in master release (not also, it will be skipped only in master if i understand it right)

**Review by kopecmartin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (son-vyas) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (son-vyas) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (son-vyas) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (son-vyas) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (son-vyas) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (son-vyas) [reply README.md:54]: In the allow list we are defining tests to be included/executed but here we are talking about skiplist, so removing the jobs will mean skipping the test everywhere as we did in openstack-tempest-skiplist earlier in tripleo

File: README.md (hunk 7 above)
Comment (son-vyas) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (kopecmartin) [reply README.md:54]: oh, i get it now

**Review by kopecmartin — APPROVED** [review]

**Pull request merged**